apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -340,6 +333,24 @@ class VotePanelContent extends React.Component {
   }
 }
 
+const StyledInfo = styled(Info.Action)`
+  div:first-child {
+    align-items: flex-start;
+  }
+`
+
+const NoTokenCost = () => (
+  <p css="margin-top: 10px">
+    Performing this action will{' '}
+    <span css="font-weight: bold">not transfer out</span> any of your tokens.
+    You’ll only have to pay for the{' '}
+    <SafeLink href="https://ethgas.io/" target="_blcank">
+      ETH fee
```

Comment:
```suggestion
      ETH gas fee
```

Reply:
cc @luisivan and @dizzypaty for copy suggestion here.